sdk/include/opentelemetry/sdk/trace/tracer.h

```diff
@@ -38,7 +37,7 @@ class Tracer final : public opentelemetry::trace::Tracer,
                   std::unique_ptr<InstrumentationScope> instrumentation_scope =
                       InstrumentationScope::Create("")) noexcept;
 
-  nostd::shared_ptr<opentelemetry::trace::Span> StartSpan(
+  std::shared_ptr<opentelemetry::trace::Span> StartSpan(
```

Comment:
I believe the correct approach is to use `nostd::` constructs in the SDK when implementing the Otel API interfaces, as specified in their declarations.